app/Http/Controllers/ForwardChatworkController.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Webhook;
+use App\Models\Condition;
+use Illuminate\Http\Request;
+
+class ForwardChatworkController extends Controller
+{
+    public function forwardMessage(Request $request, $token)
+    {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.PHP.LowerCaseConstant.Found</br>**Severity**: error</br>**File**: app/Http/Controllers/ForwardChatworkController.php</br>**Line**: 14</br>TRUE, FALSE and NULL must be lowercase; expected "false" but found "FALSE"